src/gc/gc.cpp

```diff
@@ -30623,34 +30500,21 @@ BOOL gc_heap::large_object_marked (uint8_t* o, BOOL clearp)
 }
 
 #if defined(GC_PROFILING) || defined(FEATURE_EVENT_TRACE)
```

Comment:
I still have checks for GC_PROFILING and FEATURE_EVENT_TRACE in gc.cpp because I don't want to include the code that does stuff only with diagnostics if those are not defined. but I am not opposed to unconditionally compile this in 'cause we almost always have FEATURE_EVENT_TRACE defined.